ibis/backends/tests/test_temporal.py

```diff
@@ -1861,18 +2147,19 @@ def test_timestamp_literal(con, backend):
     ["oracle"], raises=sa.exc.DatabaseError, reason="ORA-00904: MAKE TIMESTAMP invalid"
 )
 @pytest.mark.parametrize(
-    ("timezone", "expected"),
+    ("timezone", "expected", "expected_for_flink"),
```

Comment:
@mfatihaktas should we be allowing different expectations for Flink? Kind of kills the portability across backends

Reply:
Yes, I agree. I think the correct result should include the timezone literal (e.g., `GMT`, `PST`) per the test function name `test_timestamp_with_timezone_literal`. However, I have not been able to find a way to get the timezone literal from Flink. We can obviously append it with `CONCAT` (after mapping the given timezone to the correct literal) but I was not sure if that would be appropriate.

Do you have further thoughts on this?

Reply:
If Flink can't produce the expected values, then it fails this test.

We can add separate tests just for Flink in the backend-specific locations

Reply:
> We can obviously append it with CONCAT (after mapping the given timezone to the correct literal) but I was not sure if that would be appropriate.

@mfatihaktas I think this seems pretty reasonable? I can mark it `notimpl` for now, but with a TODO to do this.

Reply:
> I can mark it notimpl for now, but with a TODO to do this.

Sounds good to me. Thanks!